pii_secret_check_hooks/pii_secret_file_content.py

```diff
@@ -20,13 +20,13 @@ def main(argv=None):
     parser.add_argument(
         "--exclude",
         nargs="?",
-        default="pii-secret-exclude.txt",
```

Comment:
Sorry! Just to be a PIA I want to use the non '.' version of the file name,

Reply:
Cool